packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -63,6 +63,7 @@ describe("Garbage Collection Tests", () => {
             { gcAllowed: true, deleteTimeoutMs },
             (unusedRoutes: string[]) => {},
             () => Date.now(),
+            () => {},
```

Comment:
You can a simple unit test for session expiry by adding a callback here and validating that the callback is invoked.